Small Sputnik V updates (#889)
greenelab/covid19-review: content/22.vaccines.md

Of these individuals, 68% experienced joint and muscle pain, 54% had injection site pain, 11% reported redness and swelling, 40% had a fever, and 5% reported diarrhea. 
Only 5% of the vaccinated participants experienced serious adverse events that required medical attention, of which one was monitored as an inpatient.

> Additionally, an Independent assessment of Sputnik V in a phase II clinical trial in India found the vaccine to be effective, but the data is not yet publicly available [https://www.reuters.com/article/health-coronavirus-india-vaccine-idUSL4N2JM2XA]. 

Reviewer: ```suggestion
Additionally, an Independent assessment of Sputnik V in a phase II clinical trial in India found the vaccine to be effective, but the data is not yet publicly available [@url:https://www.reuters.com/article/health-coronavirus-india-vaccine-idUSL4N2JM2XA]. 
```

On December 21st, 2020, Gamaleya, AstraZeneca, R-Pharm, and the Russian Direct Investment Fund agreed to assess the safety and immunogenicity of the combined use of components of the AstraZeneca and University of Oxford AZD1222 (ChAdOx1) vaccine and the rAd26-S component of the Sputnik V vaccine in clinical trials [@url:https://rdif.ru/Eng_fullNews/6220/].

> This agreement hopes to establish scientific and business relations between the entities with an aim to co-developing a vaccine with long-term immunization. 

Reviewer: ```suggestion
This agreement hopes to establish scientific and business relations between the entities with an aim to co-develop a vaccine providing long-term immunization. 
```

The trial, which will begin enrollment soon, will include 100 participants in a phase II open-label study and is hoped to be complete within 6 months. 
Participants will first receive an intramuscular dose of AZD1222 on day 1, followed by a dose of rAd26 on day 29. 
Participants will be monitored from day 1 for 180 days in total. 
The primary outcomes measured will include incidence of serious adverse events post first dose until the end of the study. 
Secondary outcome measures will include incidence of local and systemic adverse events 7 days post each dose, a time course of antibody responses for the Spike protein and the presence of anti-SARS-CoV-2 neutralizing antibodies [@clinicaltrials:NCT04686773].

Overall, there is hesitancy surrounding the management of the Sputnik V vaccine approval process and concerns over whether the efficacy data may be inflated due to a lack of asymptomatic testing within the trial.

> However, the results of the phase III study were promising and further trials are underway, which will likely shed light on the overall efficacy and safety of the Sputnik V vaccine regimen. 

Author: ```suggestion
However, the interim results of the phase III study were promising and further trials are underway, which will likely shed light on the overall efficacy and safety of the Sputnik V vaccine regimen. 
```

> However, the results of the phase III study were promising and further trials are underway, which will likely shed light on the overall efficacy and safety of the Sputnik V vaccine regimen. 

Author: ```suggestion
However, the interim results of the phase III study were promising and further trials are underway, which will likely shed light on the overall efficacy and safety of the Sputnik V vaccine regimen. 
```
